cmd/tk/workflow.go

```diff
@@ -21,6 +21,14 @@ const (
 	ExitStatusDiff = 16
 )
 
+func validateDryRun(dryRunStr string) error {
+	switch dryRunStr {
+	case "", "none", "client", "server":
+		return nil
+	}
+	return fmt.Errorf("--dry-run must be either: \"\", \"none\", \"server\" or \"client\"")
```

Comment:
Nit: to avoid escaping `"` you could do:
```suggestion
	return fmt.Errorf(`--dry-run must be either: "", "none", "server" or "client"`)
```

Something similar can be done in the lines below.

Reply:
Thanks for the suggestion, applied :+1: